terraform/workspace-variables/production.tfvars

```diff
@@ -5,7 +5,7 @@ paas_space_name                  = "bat-prod"
 paas_postgres_service_plan       = "tiny-unencrypted-11"
 paas_redis_service_plan          = "tiny-ha-5_x"
 paas_app_start_timeout           = "180"
-paas_web_app_deployment_strategy = "blue-green-v2"
+paas_deployment_strategy         = "blue-green-v2"
```

Comment:
We can remove this variable as we don't set it to different values across the environment